### PR TITLE
dark theme feature

### PR DESCRIPTION
**Pull request by lasgra**
added new option in settings - dark mode
dark mode colors look bad, but I'm not a UI designer and someone (michał) will change colors later
also fixed account table
image

image

image

image

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 @Component({
   selector: 'pp-root',
   templateUrl: './app.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AppComponent {
   private spinnerService = inject(SpinnerService);
+  private settingsService = inject(SettingsService);
+
+  constructor(private renderer: Renderer2) {}
```
Comment (Majkoo) [inline src/app/app.component.ts]: please stick to one dependency injection pattern

File: src/app/shared/components/header/header.component.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
             Welcome back, <br> <b>{{acc.nickname}}</b>
           </p>
           <img
-            class="rounded-full w-14 h-14 aspect-square p-2"
+            class="rounded-full w-10 h-10 mx-2 aspect-square bg-grey dark:bg-dark-grey"
```
Comment (Majkoo) [inline src/app/shared/components/header/header.component.html]: rm that bg-grey if it wasnt there before

File: src/app/shared/components/like-btn/like-btn.component.html [hunk 3]
```diff
@@ -10,5 +10,6 @@
   }"
   [@likedState]="pic.isLiked"
   (click)="like(pic.id)"
+  class="text-dark dark:text-dark-dark"
```
Comment (Majkoo) [inline src/app/shared/components/like-btn/like-btn.component.html:13]: what does text-dark-dark mean?

Please review the names yourself. Here I suppose its an inversion of "text-dark" which if I am right should be "text-light"

Comment (Majkoo) [reply src/app/shared/components/like-btn/like-btn.component.html:13]: We'll have to change color scheme to be a bit more flexible, let it be the first step

File: src/app/shared/components/header/header.component.html (hunk 2 above)
Comment (lasgra) [reply src/app/shared/components/header/header.component.html]: but it needs to change color to be fitting background

File: src/app/app.component.ts (hunk 1 above)
Comment (lasgra) [reply src/app/app.component.ts]: ?

File: src/app/shared/components/like-btn/like-btn.component.html (hunk 3 above)
Comment (lasgra) [reply src/app/shared/components/like-btn/like-btn.component.html:13]: we literally talked about how to name dark theme colors
image

**Pull request merged**